themes/custom/droopler_theme/scss/layout/_after_footer.scss

```diff
@@ -0,0 +1,30 @@
+//Styles for the menu placed after the main footer.
+
+//Color variables used in this component.
+$after-footer-background-color: $color-hecuba !default;
+$after-footer-text-color: $color-odysseus !default;
+
+.after-footer {
+
+  //Menu block from d_demo
+  .region-footer-main {
+    background-color: $after-footer-background-color;
+
+    ul.nav {
+      display: flex;
+      justify-content: space-between;
+      align-items: center;
+      width: 50%;
```

Comment:
a ten, moze to tez jako zmienna ?

Reply:
A czemu tu jest 50%?
Może tu jest potrzebny container?

Reply:
chyba w layoucie wygladalo jak by bylo male.. ale mozliwe ze to tylko kwestia tresci - zeby byla wysrodkowana ?
![image](https://user-images.githubusercontent.com/37533838/72138937-2063b780-338e-11ea-982a-670a7dcc4b3b.png)

@grzegorz-pietrzak-droptica  @piotr-kamieniecki-droptica

Reply:
Wsadźmy całość w container i wyśrodkujmy